### PR TITLE
telegram-notify 2.0.0

### PR DESCRIPTION
**Pull request by mcaldas**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by mcaldas** [comment]
any update here?

File: steps/telegram-notify/2.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  telegram_chat_id: $TELEGRAM_CHAT_ID
+- download_url: $DOWNLOAD_URL
+  opts:
+    description: |
+      The url where artifacts can be downloaded.
+    is_required: true
+    title: Download URL
+- custom_message: $CUSTOM_MESSAGE
+  opts:
+    description: "Any info you would like to append to the message. \nIt is sent both
```
Comment (godrei) [inline steps/telegram-notify/2.0.0/step.yml:63]: NOTE: `\n` will not cause a line break in the Workflow Editor, you can use two spaces at the end of the line to achieve this. I suggest updating the step.yml in a new step version.

You can verify the step.yml locally by running: `bitrise :workflow-editor`, if you have a workflow that uses the current [step from a local directory path](https://devcenter.bitrise.io/en/references/steps-reference/step-reference-id-format.html#the-path---source).

**Review by godrei — APPROVED** [review]

**Comment by godrei** [comment]
Hi @mcaldas ,
thank you for the new step version!
I left a suggestion for the next version.

File: steps/telegram-notify/2.0.0/step.yml (hunk 1 above)
Comment (mcaldas) [reply steps/telegram-notify/2.0.0/step.yml:63]: thank you @godrei! will definitely look into that for next version

**Pull request merged**